pkgs/applications/networking/mailreaders/mailpile/default.nix

```diff
@@ -22,18 +23,27 @@ python2Packages.buildPythonApplication rec {
     appdirs
```

Comment:
Can we use python3 instead?

Reply:
No, Mailpile does not support python 3 (they are planning it since 2014). This "updated" version is from 2019 (last release on GitHub) so there is also no python 3 support.

Reply:
I found a similar pull request updating Mailpile (https://github.com/NixOS/nixpkgs/pull/97274). However, the last pull request does not work anymore because of the `fasteners` library. So when this is merged, the old PR can be closed.
Also it seems like you already asked whether Python 3 support is possible one year ago :D

Reply:
Python2 should be removed from as much as possible because we no longer maintain it and don't even check for breakages when updating libraries.